qiskit_experiments/library/characterization/cr_hamiltonian.py

```diff
@@ -0,0 +1,472 @@
+                bind_circuit.metadata["dt"] = dt_factor
+
+                expr_circs.append(bind_circuit)
+
+        return expr_circs
+
+
+class EchoedCrossResonanceHamiltonian(CrossResonanceHamiltonian):
+    r"""Echoed cross resonance Hamiltonian tomography experiment.
+
+    # section: overview
+
+        This is a variant of :py:class:`CrossResonanceHamiltonian`.
+        The experiment framework is identical to it, however, in this experiment,
+        cross resonance tone is realized with the echoed cross resonance.
+
+        .. parsed-literal::
+
+                 ┌──────────────────────┐┌───┐┌──────────────────────┐
+            q_0: ┤0                     ├┤ X ├┤0                     ├
+                 │  cr_tone_p(duration) │└───┘│  cr_tone_m(duration) │
+            q_1: ┤1                     ├─────┤1                     ├
+                 └──────────────────────┘     └──────────────────────┘
+
+        Where the ``cr_tone_p`` and ``cr_tone_m`` are the cross resonance tone with
+        positive and negative pulse amplitude. The net Hamiltonian of the gate
+        eliminates IX and ZI interactions [1]. Note that the CR Hamiltonian tomography experiment
+        cannot detect the ZI term, however it is sensitive to the IX term.
+
+        If you are only interested in the ZX term as a generator of the CNOT gate,
+        sometime this pulse sequence can improve the fitting performance within
+        the limited available hardware resource.
+        The ZX + IX induces control state dependent Rabi oscillation on the target qubit
+        and its oscillation rate will be pretty different when these interaction strengths
+        are close to each other. In this case, you may need fine duration scan
+        while keeping longer maximum duration to get enough oscillation cycles for fitting.
+        This situation often causes the waveform memory overflow in the control electronics.
+        The elimination of IX term by echoing may save us from such a dilemma.
+
+    # section: reference
+        .. ref_arxiv:: 1 2007.02925
+
+    """
+
+    def _define_decomposition(
+        self,
+        backend: Backend,
+        duration: circuit.Parameter,
+        sigma: float,
+    ) -> QuantumCircuit:
+        """Echoed cross resonance.
+
+        Args:
+            backend: The target backend.
+            duration: Parameter object representing a duration of cross resonance pulse.
+            sigma: Sigma of Gaussian edges.
+
+        Returns:
+            A circuit decomposition for the cross resonance to measure.
+        """
+        cr_gate_p = circuit.Gate("cr_tone_p", num_qubits=2, params=[duration])
+        cr_gate_m = circuit.Gate("cr_tone_m", num_qubits=2, params=[duration])
+
+        cr_circuit = QuantumCircuit()
+        cr_circuit.add_register(2)
+        cr_circuit.append(cr_gate_p, [0, 1])
+        cr_circuit.x(0)
+        cr_circuit.append(cr_gate_m, [0, 1])
```

Comment:
I'm wondering if we can save ourselves some schedule gymnastics by exposing as much as possible in the circuit? I.e. to get `cr90m` without a schedule could we do:
```
circ.append(cr_gate_p, [0,1])
circ.x(0)
circ.rz(np.pi, 0)
circ.rz(np.pi, 1)
circ.append(cr_gate_p, [0,1])
circ.rz(np.pi, 0)
circ.rz(np.pi, 1)
```

Reply:
Good point. Actually you don't need `circ.rz(np.pi, 0)` because control channel frame is associated with the target qubit. But I'm not 100% confident on if the backend behaves as I'm expecting. @wshanks do you think this works?

Reply:
I tested above decomp with `ibmq_sydney` and confirmed it works. Please check update in https://github.com/Qiskit/qiskit-experiments/pull/385/commits/369a0ec13e07a5b83e79e9fd324725cde09e3494

Reply:
Code looks OK to me. If this does what it should on the backend then its fine.